challenges/objects-arrays.js

```diff
@@ -42,24 +65,39 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":9,"first_name":"Michail","university":"Universidad Católica de Ávila","email":"mrome8@shinystat.com"},
 {"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
 
-/* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
+/* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+for (i=0; i<graduates.length; i++){
+  universities.push(graduates[i].university)
+  universities.sort()
+}
 console.log(universities)
 
-/* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
+/* Request 2: Create a new array called contactInfo that contains both first name
+and email of each student.
 
-The resulting contact information should have a space between the first name and the email information like this: 
+The resulting contact information should have a space between the first name and
+the email information like this:
 Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (i=0; i<graduates.length; i++){
```

Comment:
Very good! Don't forget to declare the variable here with `let`, such as `let i = 0`. For convention, put a space between each operator: `let i = 0; i < graduate.length; i++`